spec/factories/twitter_accounts.rb

```diff
@@ -7,15 +7,14 @@
     user_secret { "ACCESS_SECRET" }
     twitter_account_info { {} }
     city { "New York" }
-    state { "NY" }
-
     address { "278 Broadway, New York, NY 10007, USA" }
     latitude { 40.7143528 }
     longitude { -74.0059731 }
 
     after(:build) do |twitter_account, _evaluator|
-      usa = Country.find_by(iso: "US") || FactoryBot.build(:country_us)
-      twitter_account.country = usa
+      new_york = State.find_by(abbreviation: "NY") || FactoryBot.build(:state_new_york)
```

Comment:
Something cool that could be done with all these location traits, now that everything has the same attributes - we could only define the traits once, in the `spec/factories.rb` file - https://stackoverflow.com/questions/23204677/share-traits-across-different-factories

And then we wouldn't need to include Los Angeles, nyc, chicago, amsterdam - anywhere else.

Reply:
Nice! good catch, thanks — updated in 3c31ff6f